lua/fzf-lua-frecency/health.lua

```diff
@@ -10,11 +11,27 @@ function M.check()
     })
   end
 
-  if vim.fn.executable "fd" == 1 then
+  if vim.fn.executable "fdfind" == h.vimscript_true then
     vim.health.ok "'fd' is installed"
+  elseif vim.fn.executable "fd" == h.vimscript_true then
+    vim.health.ok "'fd' is installed"
+  elseif vim.fn.executable "rg" == h.vimscript_true then
+    vim.health.ok "'rg' is installed"
+  elseif vim.fn.executable "find" == h.vimscript_true then
+    vim.health.ok "'find' is installed"
   else
-    vim.health.error("'fd' is not installed", {
+    vim.health.error("'all_files' requires 'fd' or 'rg' to be installed", {
       "Install fd: https://github.com/sharkdp/fd",
+      "Install rg: https://github.com/BurntSushi/ripgrep",
+      "Install find: https://www.gnu.org/software/findutils/",
+    })
+  end
+
+  if vim.fn.executable "awk" == h.vimscript_true then
+    vim.health.ok "'awk' is installed"
+  else
+    vim.health.error("'awk' is not installed", {
+      "On windows, one installation method for 'awk' is through the choco package manager: https://community.chocolatey.org/packages/awk",
```

Comment:
To support windows we probably need to change the all_filed combined command syntax, the separator between the commands needs to be `&` instead of `;`, I’m also not sure if Windows supports wrapping with `()`, I can’t tret it as I no longer have access to a windows VM but I do most of fzf-lua Windows testing via the CI.

Reply:
Huh interesting, there's more differences than I realized, this section isn't very helpful then :sweat_smile: